src/main/resources/hbase-site.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+/**
+* #%L
+* OBKV HBase Client Framework
+* %%
+* Copyright (C) 2024 OceanBase Group
+* %%
+* OBKV HBase Client Framework  is licensed under Mulan PSL v2.
+* You can use this software according to the terms and conditions of the Mulan PSL v2.
+* You may obtain a copy of Mulan PSL v2 at:
+*          http://license.coscl.org.cn/MulanPSL2
+* THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+* EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+* MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+* See the Mulan PSL v2 for more details.
+* #L%
+*/
+-->
+
+<configuration>
```

Comment:
it's better to add this file to src/main/test/resources, because user may use obkv-hbase-client and native-hbase-client simultaneously

Reply:
Already moved from src/main/resources to src/test/resources.